hcl2template/testdata/variables/basic.pkr.hcl

```diff
@@ -29,6 +29,7 @@ variable "super_secret_password" {
 description = <<IMSENSIBLE
```

Comment:
Did you mean "sensitive" here instead of "sensible"?

Reply:
yes !